docs/en_US/Tutorial/Nnictl.md

```diff
@@ -319,6 +319,48 @@ Debug mode will disable version check function in Trialkeeper.
   |------|------|------ |------|
   |id|  False| |ID of the experiment you want to set|
 
+* __nnictl trial head__
+
+  * Description
+  
+    You can use this command to show trial results with highest metric. Shown results are sorted by default metric decreasing.
+
+  * Usage
+
+    ```bash
+    nnictl trial head [OPTIONS]
+    ```
+
+  * Options
+
+  |Name, shorthand|Required|Default|Description|
+  |---|---|---|---|
+  |id|  False| |ID of the experiment you want to set|
+  |--num, -n|False|10|the number of listed items|
+
+  Note that if `--num` exceeds the number of trials, all trial results are listed.
+
+* __nnictl trial tail__
```

Comment:
suggest to merge `tail` and `head` into `nnictl trial ls`. what do you think? @chicm-ms @SparkSnail

Reply:
agree, `nnictl trial ls` show all of trial results, and this `nnictl trial tail` filter trial results, we can merge them to `nnictl trial ls --tail {number}` to simplify `nnictl`.

Reply:
My concern is that `nnictl trial ls` shows too much information and it is displayed in JSON format. Users cannot see the desired results at first glance even if trial results are sorted. Can we show the information in tables?

Reply:
@tabVersion i don't get. what do you mean?